projects/storefrontlib/src/cms-components/user/login/login.component.ts

```diff
@@ -22,5 +52,21 @@ export class LoginComponent implements OnInit {
         }
       })
     );
+
+    this.subscription = this.routingService
+      .getRouterState()
+      .subscribe(routerState => {
+        if (routerState.state.context.id.indexOf('/checkout/') !== -1) {
```

Comment:
So I've been thinking about this. I'm definitively not a fan of using a string for the url. We have a configuration with the different urls. Using `this.routingConfigService.getRouteConfig('checkout').paths[0];` you can get the path associated for the checkout key.

I think it's a better solution.